README.md

```diff
@@ -8,30 +8,107 @@ Cargo subcommands that build and test all feature flag combinations for a crate.
 
 ```
 cargo install cargo-all-features
+
+# or via cargo-binstall
+cargo binstall cargo-all-features
 ```
 
+
 ## Usage
 
 The following commands can be run within a Cargo package or at the root of a Cargo workspace.
 
 Build crate with all feature flag combinations:
 
 ```
-cargo build-all-features <CARGO BUILD FLAGS>
+cargo all-features build -- <CARGO BUILD FLAGS>
 ```
 
 Check crate with all feature flag combinations:
 
 ```
-cargo check-all-features <CARGO CHECK FLAGS>
+cargo all-features check -- <CARGO CHECK FLAGS>
 ```
 
 Test crate with all feature flag combinations:
 
 ```
-cargo test-all-features <CARGO TEST FLAGS>
+cargo all-features test -- <CARGO TEST FLAGS>
+```
+
+<details>
+    <summary markdown="title"><bold>Supported tools</bold></summary>
+
+- First party
+    - [`cargo test`](https://doc.rust-lang.org/cargo/commands/cargo-test.html) cargos integrated testing tool
+    - [`cargo check`](https://doc.rust-lang.org/cargo/commands/cargo-check.html) cargos integrated checking tool
+    - [`cargo build`](https://doc.rust-lang.org/cargo/commands/cargo-build.html) cargos integrated build tool
+    - `cargo bench` [Used by cargos benching feature](https://doc.rust-lang.org/cargo/commands/cargo-bench.html) or crates like [citerion](https://github.com/bheisler/criterion.rs)
+- Additional RustUp components
+    - [`cargo miri test`](https://github.com/rust-lang/miri) for testing using miri -> _rustup component `miri` is needed_
+- Cargo plugins
+    - [`cargo udeps`](https://github.com/est31/cargo-udeps) to analyze for unused dependencies -> _cargo plugin `cargo-udeps` is needed_
+    - [`cargo tarpaulin`](https://github.com/xd009642/tarpaulin) to analyze for unused dependencies -> _cargo plugin `cargo-tarpaulin` is needed_
```

Comment:
`tarpaulin` is a code coverage reporting tool, and seems like not a unused dependencies analyzer:

> **Tarpaulin** is a **code coverage reporting tool** for the Cargo build system, named for a waterproof cloth used to cover cargo on a ship. Currently, tarpaulin provides working line coverage and while fairly reliable may still contain minor inaccuracies in the results. A lot of work has been done to get it working on a wide range of projects, but often unique combinations of packages and build features can cause issues so please report anything you find that's wrong. Also, check out our roadmap for planned features.

Reply:
Sry, yes totally missed that copy paste error, thanks